src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -102,17 +108,15 @@ public boolean isReturnStatus() {
 
     public abstract static class DurableTaskStepDescriptor extends StepDescriptor {
 
-        public static final String defaultEncoding = "UTF-8";
-
         public FormValidation doCheckEncoding(@QueryParameter boolean returnStdout, @QueryParameter String encoding) {
+            if (encoding.isEmpty()) {
```

Comment:
Check for null just in case the form is messed up. Ideally it makes sense to use `StringUtils.isBlank()` here and in the constructor

Reply:
If the form is messed up, there is a product bug and validation will fail. No need to be unreasonably defensive.

Reply:
(for #64)